config.minimal.yml

```diff
@@ -14,3 +14,157 @@ session:
   # The domain to protect.
   # Note: Authelia must also be served by that domain.
   domain: example.com
+
```

Comment:
I think the tests are broken due to this change and not reflected in the tests.

Reply:
Sorry, which change exactly do you mean here? The empty line?

Reply:
Currently travis is failing at the minimal docker config:

[ENVIRONMENT] Running: docker-compose -f docker-compose.minimal.yml -f example/compose/docker-compose.base.yml -f example/compose/nginx/minimal/docker-compose.yml down
  1 passing (25s)
  4 failing

  1) Fail TOTP challenge
       successfully login as john
         "before all" hook:
     Error: ENOENT: no such file or directory, open '/tmp/authelia/notification.txt'

Reply:
Seems it was indeed the commented out filesystem notification that made the travis check fail.